src/cli/build.rs

```diff
@@ -11,6 +11,9 @@ pub struct Build {
     /// The app configuration file.
     #[clap(parse(from_os_str), long, default_value = DEFAULT_CONFIG_FILE)]
     pub config: std::path::PathBuf,
+
+    #[clap(parse(from_os_str), parse(try_from_str), default_value = "false")]
```

Comment:
Use boolean option instead of string:

```suggestion
    #[clap(long, action)]
```